### ISSUE TITLE
Too Many Inactive Players In CTB

### ISSUE THREAD

**Issue by TomyDoesThings** [ppy/osu-web#9782]
To put it shortly, there seems to be too many inactive players in CTB which affects ranking clarity. Them getting removed should be considered.

At the time I am typing this, there are 138 inactive players in the top 1K of CTB.

Meanwhile, for standard - 26, taiko - 57, and (all) mania - 59.
(note: I got all this information with a Java program I made.)

I can provide further information for the top 1K of each mode if needed.

In September and October of last year, all modes except CTB got a rework which meant clearing up the leaderboards from inactive players again. Unfortunately, with the CTB rework still rigorously being worked on, this opportunity never came. There are players that are still in the top 1K being shown after nearly 2 years of being inactive which I think is unfair for the active players in the CTB community.
![image](https://user-images.githubusercontent.com/119363392/214158134-d5867a0a-6e44-4f9a-b99d-82caffe38753.png)

A user that goes by the name just7nz mentioned this briefly but did not elaborate (https://github.com/ppy/osu-web/issues/9365#issuecomment-1280753166). This is me elaborating on it.

I propose then that to make CTB as clean from inactive players as the other modes to do the same for it now before the rework comes out. Besides, I was under the impression that inactive players eventually got removed from leaderboards, but I think I am wrong.

I am primarily mentioning all this and giving that suggestion of removing all inactive players in CTB as soon as possible (before rework) as ranking medals are becoming increasingly more difficult, but let us focus on the top 1K medal, Approaching the summit, as it is notably the hardest one.
![image](https://user-images.githubusercontent.com/119363392/214158267-5005e90d-d8ec-46bb-9922-3d57494fb25c.png)

A player that got this medal 12 months ago had a much easier time and less skill than players who got the medal this month. In other words, while a player now can be better than that player from 12 months ago, under the medal criteria, the player now cannot achieve the medal with that same skill. Removing all these inactive players will cut a lot of players slack from missing out on getting the medal way earlier.
The medal has been out ever since April 20, 2016, meaning rankings has had time to evolve for around 7 years.

To summarize,
I believe nobody would have complaints with removing all inactive players in CTB as it happened in the other 3 modes and by removing all inactive players now, people that actually deserve their ranking medals will get them. That may sound selfish, but I am just stating that leniency should be put on the active players in the CTB leaderboards.

Chances are CTB will continue getting some more inactive players and some of the only competition that will be happening before this rework coming out at an unknown or undefined time is with players that have not touched this game in forever.

Please really consider this.
